modules/network-observability-configuring-options-overview.adoc

```diff
@@ -7,10 +7,21 @@
 = Configuring advanced options for the Overview view
 You can customize the graphical view by using advanced options. To access the advanced options, click *Show advanced options*.You can configure the details in the graph by using the *Display options* drop-down menu. The options available are:
```

Comment:
You didn't change this part, but in looking at the preview, there's a missing space after "click Show advanced options."